src/styles/main.scss

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
 }
+
+$time: 0.6s;
+$wite: #eee;
+$black: #000;
+$pink: #ffb6c1;
```

Comment:
It's better to give variable names based on their function, not exact color, for example "hover-color" or "secondary-color", etc. This way if in future it is desided to change pink to some other color, you won't have to change variable name every time